megaparsack-doc/scribblings/megaparsack/reference.scrbl

```diff
@@ -242,6 +242,15 @@ Returns a parser that parses a single character that is in @racket[alphabet].
 
 @history[#:added "1.2"]}
 
+@defthing[any-char/p (parser/c char? char?)]{
+Returns a parser that parses a single character.
+                                                            
+@(parser-examples
+  (eval:check (parse-result! (parse-string (any-char/p) "µ")) #\µ)
+  (eval:error (parse-result! (parse-string (any-char/p) ""))))
+
+@history[#:added "1.2"]}
```

Comment:
I’d bump the version to `1.3` and use that here so that someone can guarantee `any-char/p` will be available by depending explicitly on that version. Otherwise, this looks good, thank you!